### PR TITLE
Add advice of using `assertEquals()` in the contributing guide

### PR DESCRIPTION
**Pull request by danicheg**
I had plans to add it within #5591 but forgot.

### CODE REVIEW

**Comment by danicheg** [comment]
Have no ideas why changing website docs broke formatting in tests.. but ok :)

File: website/src/hugo/content/contributing.md [hunk 1]
```diff
@@ -192,9 +192,12 @@ Apache License 2.0.
 * We encourage the addition of arbitrary instances to
   `org.http4s.testing.Http4sArbitraries` to support richer property
   testing.
+* For assertions in tests using `assertEquals()` more preferable than `assert()`.
```
Comment (bplommer) [inline website/src/hugo/content/contributing.md:195]:
```suggestion
* For assertions in tests using `assertEquals(a, b)` is preferable to `assert(a == b)`. 
```

**Review by bplommer — APPROVED** [review]
Minor nitpick but LGTM!

File: website/src/hugo/content/contributing.md (hunk 1 above)
Comment (danicheg) [reply website/src/hugo/content/contributing.md]: Thanks!

**Pull request merged**